src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -6139,7 +6139,7 @@ public class TypeDelegator : System.Reflection.TypeInfo
     public delegate bool TypeFilter(Type m, Object filterCriteria);
     public abstract partial class TypeInfo : System.Type, System.Reflection.IReflectableType
     {
-        internal TypeInfo() { }
+        protected TypeInfo() { }
```

Comment:
@danmosemsft - The desktop does not expose a protected constructor for TypeInfo and adding one now opens the door to the user-defined TypeInfo's. Has this addition been approved by the api-review board?